app/src/ui/branches/pull-request-badge.tsx

```diff
@@ -15,18 +15,21 @@ export class PullRequestBadge extends React.Component<
   IPullRequestBadgeProps,
   {}
 > {
-  public render() {
+  private renderCIStatus(): JSX.Element | null {
     const status = this.props.status
 
     if (!status || status.totalCount === 0) {
       return null
     }
 
+    return <CIStatus status={status} />
+  }
+
+  public render() {
     return (
       <div className="pr-badge">
         <span className="number">#{this.props.number}</span>
-
-        <CIStatus status={status} />
+        {this.renderCIStatus()}
```

Comment:
```ts
status && status.totalCount ? <CIStatus status={status} /> : null
```
?

Reply:
I would’ve suggested `status && status.totalCount && <CIStatus status={status} />`, but then the `0` count would get rendered as a literal “0” in the app.